app/build.gradle

```diff
@@ -275,6 +275,7 @@ dependencies {
     implementation ("com.github.Outblock:flow-jvm-sdk:v0.5.7") {
         exclude group: 'org.bouncycastle', module: 'bcprov-jdk15on'
     }
+    implementation 'com.github.onflow.flow-kmm:flow:0.0.1'
```

Comment:
0.0.1 is not the latest version

Reply:
Please use 0.0.2 instead
https://github.com/onflow/flow-kmm/releases/tag/0.0.2
https://github.com/onflow/flow-kmm/actions/runs/12783494611